tx/textual/valuerenderer/message.go

```diff
@@ -79,6 +98,73 @@ func (mr *messageValueRenderer) Format(ctx context.Context, v protoreflect.Value
 	return screens, nil
 }
 
+func (mr *messageValueRenderer) FormatRepeated(ctx context.Context, vr ValueRenderer, v protoreflect.Value, name string) ([]Screen, error) {
+	l := v.List()
+
+	if l == nil {
+		return nil, fmt.Errorf("non-List value")
+	}
+
+	screens := make([]Screen, 1)
+
+	screens[0].Text = fmt.Sprintf("%d %s", l.Len(), formatPluralFieldKind(l.Len(), vr.Kind()))
```

Comment:
```suggestion
	screens[0].Text = fmt.Sprintf("%d %s", l.Len(), formatPluralFieldKind(l.Len(), getKind(fd)))
```

where getKind is defined in this commit: https://github.com/cosmos/cosmos-sdk/commit/72d8d7243b6facb07ab626acd9bf6f1e9561bc97

```go
// getKind returns the field kind: if the field is a protobuf
// message, then we return the message's name. Or else, we
// return the protobuf kind.
func getKind(fd protoreflect.FieldDescriptor) string {
	if fd.Kind() == protoreflect.MessageKind {
		return string(fd.Message().Name())
	}

	return fd.Kind().String()
}
```

Reply:
For decimals, we'll probably get `3 Strings` instead of `3 sdk.Decs` like in your `Kind()` method proposal, but I think that's fine

Reply:
Curious how that will work with Parse 🤔